src/DataIntegrity.php

```diff
@@ -105,6 +108,54 @@ public static function ensureFieldsPresent(dict<string, mixed> $row, table_schem
               } else {
                 $row[$field_name] = (int)$row[$field_name];
               }
+            } else {
+
+                $signed = !($field_unsigned);
+                $field_value = (int)$row[$field_name];
+                $row[$field_name] = $field_value;
```

Comment:
is this bit necessary?

Reply:
Line 114 is necessary since the type checker assumes row[field_name] is mixed, do you think the field_value variable isnt necessary ?

Reply:
I agree 114 is necessary, but 115 seems to be a no-op since we've already tested the type and we know it's an int.